### ISSUE TITLE
Youtube ads are not being clicked/collected

### ISSUE THREAD

**Issue by dave90dave** [dhowe/AdNauseam#2465]
### What type of Ad is not being collected

- Video
- Mid-roll Video Ad

### URL(s) where the ads appear.

www.youtube.com

### Your location/country.

netherlands

### Screenshot(s) of uncollected ads (disable AdNauseam if they are being correctly hidden)

none are being blocked

**Comment by dhowe** [comment]
ads are not being blocked, or not being clicked ?

**Comment by dave90dave** [comment]
Hi @dhowe 
It blocks the ads correctly, but it doesn't click them.

Also the ads on the side of the video are not being clicked. (but are correctly being blocked)

**Comment by dave90dave** [comment]
Here you see the result with the adnouseam clicker enabled, and adblockers disabled:

![Schermafbeelding 2023-11-15 123031](https://github.com/dhowe/AdNauseam/assets/7985017/18b9ff13-3790-4e79-9ae5-a6b0c7703f76)


It works fine on other websites.

**Comment by dave90dave** [comment]
And I have the same problem with Adsense ads on third party websites

**Comment by dave90dave** [comment]
This is google adsense on www.funda.nl
Ad blocking works, but collecting and clicking doesn't.

![Schermafbeelding 2023-11-15 150217](https://github.com/dhowe/AdNauseam/assets/7985017/ebed529d-6e00-4f8c-84ea-92079bbcfb98)

**Comment by mneunomne** [comment]
Sadly it is very hard to collect ads from youtube, even if we implement the parser, it changes so often that it's almost impossible to keep up. Plus the fact that the method of ad blocking on youtube usually are not cosmetic, but rather by filtering requests, and overwriting functions with scriptlets.

**Comment by mneunomne** [comment]
> This is google adsense on [www.funda.nl](http://www.funda.nl/)

I'll take a look on this one.

**Comment by dave90dave** [comment]
Hi @mneunomne 

I tried to reproduce it again, with version Adnausem 3.19.0 (chromium package)
I found the following:
* The extension is not working well in the Brave browser (which is chromium). Probably it somehow conflicts with the built in ad-blocker that can not be removed.
* When I use microsoft edge, it only clicks ads correctly when I enable 'Hide ads'. If I don't enable 'Hide ads' it will show the ads, but will not click them. (edge is chromium based too)
